### PR TITLE
Point renovate at our pip requirements files

### LINKED ISSUE #14088: fix/replace pyup
we're 6 days without any new pyup.io pull requests being created and my https://pyup.io/account/account/ doesn't show any mozilla organisation reps any longer.

It's unclear if it's a change on our side to restrict access (the application permissions page show ❌ next to organisation access) or on their side (the account page on pyup says "Please upgrade your account to the Business or Organization plan to allow organization access." - I don't know if that's new and we need to upgrade something or it's always said that and it only applies to private/commercial projects)

The alternative to fixing pyup is switching it out for another tool/service.

### PR DESCRIPTION
**Pull request by bobsilverberg**
Fixes #14088

### CODE REVIEW

**Comment by bobsilverberg** [comment]
I guess we need to merge this to see if it actually works.

File: renovate.json [hunk 1]
```diff
@@ -19,5 +19,10 @@
...
     }
-  ]
-}
+  ],
+  "pip_requirements": {
+    "fileMatch": [
+      "^requirements/.*\\.txt$"
+    ]
+  }
+}
```
Comment (eviljeff) [inline renovate.json:28]: 🐼

Comment (bobsilverberg) [reply renovate.json:28]: I'm not sure how to interpret that panda.

Comment (willdurand) [reply renovate.json:28]: I think it is the panda that watches for trailing newline removals.

Comment (bobsilverberg) [reply renovate.json:28]: I noticed the removal, and added it back, but prettier removed it again when I saved the file, so I figured I wouldn't argue with prettier.

Comment (willdurand) [reply renovate.json:28]: Can you argue with a panda? 😛

Comment (eviljeff) [reply renovate.json:28]: Pandas love newlines at the end of files!  (in python you get a linting fail if you don't end a file with a new line)

**Pull request merged**